### PR TITLE
Update documentation for IPFS provider filebase for use on gitbook

### PR DESCRIPTION
**Pull request by mattsmithies**
## Overview

This is a small PR that at the documentation for filebase into the docs folder, specifically for reading use in Gitbook.

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by mattsmithies** [comment]
@anvabr Here's the updated docs, I noticed going through the code that the way I initially created this provider mechanism has changed somewhat to not be flexible for any IPFS provider.

Specifically, how the constructor signature for the _IpfsClientClass_ has now been overloaded with an additional key for filebase. Unfortunately, the implication of this is that your core class would need to be reworked to allow other implementations of clients as the class itself is dependent on multiple providers that will not be present at the same time for a given deployment of Guardian. 

In other words, the constructuor for _IpfsClientClass_ expects keys for both web3Storage and filebase for any instance of said client.

If you require some support to design this to be more generic and to potentially integrate interfaces/contracts, please let me know.
 
Screenshot 2024-04-15 at 15 13 15
